src/vs/vscode.d.ts

```diff
@@ -5661,6 +5661,14 @@ declare module 'vscode' {
 		 */
 		clear(): void;
 
+		/*
+		 * Replaces the existing contents of the channel with the given value.
+		 *
+		 * *Note*: this method should only be used by extensions with smaller output
+		 * channel text sizes. Use of `append` methods is preferred.
+		 */
+		replaceAll(value: string): void;
```

Comment:
Because the API is proposed its definition must go into `vscode.proposed.d.ts`

Reply:
When doing that, wrap things into a `#region`-comment (copy what other proposals have done)

Reply:
Done